index.html

```diff
@@ -109,63 +96,38 @@ <h1 class="title">
             </div>
         </div>
 
-        <p>
-            &nbsp;<br>
-            ... these are just a few examples, and this list is not meant to be exhaustive. There is more out there 🚀
-        </p>
     </div>
 
     <div class="box hero is-warning">
 
         <h1 class="title">
-            For Developers
+            for developers
         </h1>
-        <p>
-            A Webxdc is <b>HTML5 content</b> wrapped into a self-contained <code>.zip</code> named <code>.xdc</code>.
-            <br><br>
-            Attached to a chat, you are already done.
-            <br><br>
-        </p>
```

Comment:
for me, these two lines would have helped a lot for general understanding what the remaining things are about.

why remove it?

Reply:
I think it's useful too, but maybe the intention is to encourage reading the documentation without getting into details.

Reply:
in the next section the "self-contained html5 app" is mentioned so it's not all lost.  Thing is that the "Try it out" section already provides enough details to understand it. I also added some emoticons. Feel free to re-style.